internal/backends/postgresql/metadata/registry_test.go

```diff
@@ -147,49 +148,6 @@ func TestCheckAuth(t *testing.T) {
 	}
 }
 
-func TestCreateDrop(t *testing.T) {
-	if testing.Short() {
-		t.Skip("skipping in -short mode")
-	}
-
-	t.Parallel()
-
-	connInfo := conninfo.New()
-	ctx := conninfo.Ctx(testutil.Ctx(t), connInfo)
-
-	r, db, dbName := createDatabase(t, ctx)
-	collectionName := testutil.CollectionName(t)
-	testCollection(t, ctx, r, db, dbName, collectionName)
-}
-
-func TestCreateLongCollectionName(t *testing.T) {
```

Comment:
@AlekSi do I understand correctly that you deleted this test because we have integration tests that test the same (`TestCollectionName`)?

Reply:
Yeah